src/main/java/com/hillel/dzhurynskyi/homework/hm4/ex1/classes/Circle.java

```diff
@@ -0,0 +1,18 @@
+package com.hillel.dzhurynskyi.homework.hm4.ex1.classes;
+
+import com.hillel.dzhurynskyi.homework.hm4.ex1.interfaces.Area;
+
+public class Circle implements Area {
+    private final double radius;
+    private static final String name = "Circle";
```

Comment:
Keyword 'final' in lines 6-7 are as a marker only. It nothing add in your case, up to you